Gordon360/Static Classes/Names.cs

```diff
@@ -29,13 +29,14 @@ public static class Resource
         public const string CHECKIN = "Info relating to a student's Academic Check-In";
         public const string SHIFT = "A shift that a student has worked";
         public const string CLIFTON_STRENGTHS = "A student's uploaded clifton strengthsfinder results";
-        public const string RECIM = "General/Admin RecIM resource";
+        public const string RECIM = "A general or admin RecIM resource";
         public const string RECIM_ACTIVITY = "A RecIM activity resource";
         public const string RECIM_SERIES = "A RecIM series resource";
         public const string RECIM_MATCH = "A RecIM match resource";
         public const string RECIM_TEAM = "A RecIM team resource";
         public const string RECIM_SPORT = "A RecIM sport resource";
-        public const string RECIM_PARTICIPANT = "RecIM Participating User (FacStaff or Student)";
+        public const string RECIM_PARTICIPANT = "A RecIM Participating User (FacStaff or Student) resource";
```

Comment:
```suggestion
        public const string RECIM_PARTICIPANT = "A RecIM participating user resource";
```
Facstaff or student is not technically accurate according to what @amos-cha told me last week about guests being participants as well

Reply:
Customers were indeed very adamant about being able to "add people" on the spot and specifically outside people specifically Conwell students.